kostyor/tests/unit/rest_api/test.py

```diff
@@ -29,14 +29,25 @@ def setUp(self):
             'cluster_id': '1234'
         }
 
+    def _make_request_and_parse_result(self,
```

Comment:
It seems a bit inconvenient to me. I think we should go either

* ensure any response from Flask is JSON (even errors, except 5xx)
* or, provide custom `Response` wrapper to Flask's `test_client` with helpful `get_json()` that returns `None` in case of error. 

in that case we are free to prepare request as we want, and to handle responses as we want either.

What do you think?